js/cards.js

```diff
@@ -114,6 +152,56 @@ export async function defineCardComponent() {
   customElements.define("tcg-card", Card);
 }
 
+function addAnimation(e){
+  // apply the animated class to start the CSS animation
+  e.target.classList.add("animated");
+}
+function updateZIndex(e){
+    e.target.style.zIndex = card_z_index;
+    // we don't want this to update again when reanimating later
+    e.target.removeEventListener("animationstart", updateZIndex);
+}
+
+function setupHover(){
+  // setup the initial event listeners for all cards
+  let cardList = document.getElementsByClassName("card-image");
+  for (let i = 0; i < cardList.length; i++) {
+    cardList[i].addEventListener("animationend", function(){
+      cardList[i].classList.remove("animated");
+    })
+    cardList[i].addEventListener("mouseover", addAnimation);
+    cardList[i].addEventListener("click", addAnimation);
+  }
+}
+export function addHover(){
```

Comment:
to distinguish this better from `setupHover`, maybe name to `removeUnclickableFromCards`?

Reply:
I've updated both `addHover` to `removeUnclickableFromCards`, and `removeHover` to `addUnclickableToCards` with a comment.